config/migrate_plus.migration.plan_files.yml

```diff
@@ -40,19 +40,19 @@ process:
   _subfolder:
     plugin: format_date
     from_format: Y-m-d
-    to_format: Y/m/
+    to_format: /Y/m/
```

Comment:
file_destination ends in / , so cat-ting this will result in `documents//Y/m/`

I'm not sure if that hurts anything, but it's probably not helping